google/cloud/pubsub/internal/tracing_exactly_once_ack_handler.cc

```diff
@@ -45,13 +45,21 @@ class TracingExactlyOnceAckHandler
       opentelemetry::nostd::shared_ptr<opentelemetry::trace::Span>
           subscribe_span)
       : child_(std::move(child)), subscribe_span_(std::move(subscribe_span)) {}
+  using Attributes =
+      std::vector<std::pair<opentelemetry::nostd::string_view,
```

Comment:
I'm not sure where these `string_views` originate, so it might be nice to mention who owns the actual data.

Reply:
Some of them are consts from semantic conventions and others are string literals. Does that help clarify? I'm not sure what to specifically address.

Reply:
Yes, if they all have global extent (and that would be obvious to anyone better informed than I), then I'm happy.  Thanks.